### ISSUE TITLE
Configs for GPT?

### ISSUE THREAD

**Issue by rexdu2003** [JonasGeiping/cramming#41]
Thanks for your great jobs! I want to compare BERT with GPT under the same model size setting, so I wonder if there are any configs for training a GPT-like model. Is it enough to just remove the mask token in the input and change the attention mask and prediction target accordingly?

**Comment by JonasGeiping** [comment]
Hi, this code-base is not really set up for autoregressive modeling, but yeah, changing the attention to be causal (btw there are remnants of a causal attention mask implemented, but I would check whether that works as expected) and changing the objective would be sufficient, alongside with changing the dataloader.

**Comment by rexdu2003** [comment]
Thanks! I will take a try.
